doc/bufexplorer.txt

```diff
@@ -510,7 +510,7 @@ CHANGE LOG                                              *bufexplorer-changelog*
     - Dave Eggum has made some 'significant' updates to this latest
       version:
       * Added BufExplorerGetAltBuf() global function to be used in the
-        user�s rulerformat.
+        users rulerformat.
```

Comment:
It was likely a "smart quote", which has no place in help texts.

Maybe you can try to edit the text once more? GitHub shows a pencil symbol at the top, but it's greyed out for me.